package.json

```
@@ -48,8 +48,8 @@
 		]
 	},
 	"engines": {
-		"npm": ">=7.0.0",
-		"node": ">=16.x <=18.x"
+		"npm": ">=8.0.0",
+		"node": ">=16.x <=19.x"
```

I don't think we ever use unstable (odd-numbered) Node.js versions

```suggestion
		"node": ">=16.x <=18.x"
```